ext/native/thin3d/thin3d_vulkan.cpp

```diff
@@ -1277,18 +1350,124 @@ uint32_t VKContext::GetDataFormatSupport(DataFormat fmt) const {
 	}
 }
 
+// Simple independent framebuffer image. Gets its own allocation, we don't have that many framebuffers so it's fine
+// to let them have individual non-pooled allocations.
```

Comment:
Yeah, I expect we'll eventually run into this, but not sure what to log exactly.